e2e_samples/parking_sensors/scripts/deploy_azdo_pipelines.sh

```diff
@@ -44,6 +44,30 @@ github_sc_name="${PROJECT}-github"
 github_sc_id=$(az devops service-endpoint list --output json |
     jq -r --arg NAME "$github_sc_name" '.[] | select(.name==$NAME) | .id')
 
+ifexistsoverwrite() {
+    declare pipeline_name=$1
+    full_pipeline_name=$PROJECT-$pipeline_name
+    
+    pipeline_id=$(az pipelines show --name "$full_pipeline_name" --output json 2>/dev/null | jq -r .id)
+
+    if [ -z "$pipeline_id" ]; then
```

Comment:
line 53 to 57 seem like debug to me, can we remove or consolidate with lines 60-65?

Reply:
It is not debug. When I was creating the function I had a situation where the pipeline returned empty, so I added there to keep the logging track in case the pipeline still does not exist and it is about to be created. Basically the purpose was logging.

Reply:
ok but lines 53 to 57 can be merged with 60 to 63: 
- lines 53-57 basically do the following: if `$pipeline_id` is null they echo a message otherwise another message
- lines 60-66 do the following: if `$pipeline_id` is NOT null they delete the pipeline, else they echo a message

you could combine in a single if-then-else block: I will add a comment to explain what I mean

Reply:
ok.. got it... Cool.

Reply:
Changing this... totally make sense

Reply:
ok. Updated this one...